fastmath.py

```diff
@@ -89,6 +117,60 @@ def check_fastmath(pkg_dir, pkg_name):
     return
 
 
+def check_hardcoded_flag(pkg_dir, pkg_name):
+    """
+    Check if all `fastmath` flags are set to a config variable
+
+    Parameters
+    ----------
+    pkg_dir : str
+        The path to the directory containing some .py files
+
+    pkg_name : str
+        The name of the package
+
+    Returns
+    -------
+    None
+    """
+    ignore = [("fastmath", "_add_assoc")]
```

Comment:
I intentionally did not change the hardcoded fastmath flag for the dummy function `fastmath._add_assoc`, and we ignore checking it in this function.

Reply:
Please ignore